ddtrace/contrib/datadog_lambda/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+The datadog_lambda integration traces timeout spans in an
```

Comment:
I am a bit confused about what is meant by "timeout spans". Does "timeout spans" refer to only the ability to submit spans when Lambda functions time out? I see in the screenshot included that there are actual timeout spans produced which led me to believe that we're generating those too (but looks like it's just updating the active span with meta data).

I don't think a user of the library would know either based on this description. Could we maybe say something like:

> The datadog_lambda integration enables traces to be sent on timeout...

?

Reply:
You're right. It was a bit confusing re-reading it. I agree this should be changed, added your suggestion, added `currently enables` just to let know that more features are yet to come.